dev/ledger.md

```diff
@@ -411,6 +412,17 @@ transaction.  This is determined by the _signature_ of a transaction:
 
    - The multisignature version _v_ (current value is 1).
 
+- A valid logic-signed transaction's signature is the _lsig_ object containing
```

Comment:
Maybe recall here what makes a logic-signed tx valid

Reply:
you mean that the program runs and returns true?

Reply:
Yes, and the logic sig itself is either signed or the hash is equal to the sender address.

At the end of this part, you can add the explanation:

A logic-signed transaction's signature is valid if:
(1) Either the logic _l_ is signed by the sender of the transaction, or the hash of the logic _l_ is equal to the sender address.
(2) And the logic _l_ evaluates to true as defined in the [TEAL documentation](TEAL.md)

(check that the link above is well formed)

Reply:
Added a couple sentences explaining what combinations of values are valid, and on the execution of a valid program.